data/assets.yaml

```diff
@@ -1,5 +1,5 @@
 bucket: nextzen-tile-assets
-datestamp: 20220426
+datestamp: 20220429
```

Comment:
NIt: but 5.1.0 was only officially released on May 4th, so bring this forward?

Reply:
updated to most recent (which I think @jeffdefacto found an issue in for the iso countries.zip)

Reply:
That zero length char definition is really odd. Opens fine in QGIS but PostGIS barfs on it (saying it doesn't support 0 length varchar fields).

Filed https://github.com/mbloch/mapshaper/issues/541 upstream.

Reply:
With https://github.com/mbloch/mapshaper/issues/541 fixed upstream, I'll update Natural Earth to v5.1.1 with the fix.

Reply:
Natural Earth 5.1.1 is now out, please update here and see if the problem persists.